### ISSUE TITLE
Development On Hold?

### ISSUE THREAD

**Issue by gld1982ltd** [giucam/orbital#31]
Will there be more development to come?

**Comment by giucam** [comment]
Yes, i'm mainly waiting on libweston to materialize, plus time is not much. However i'm using orbital every day so it's not like it will rot.

**Comment by gld1982ltd** [comment]
Cool. Thanks for the reply.
